components/library/freetype/Makefile

```diff
@@ -49,6 +48,10 @@ CONFIGURE_OPTIONS+= --with-png
 CONFIGURE_OPTIONS+= --with-harfbuzz
 CONFIGURE_OPTIONS+= --with-pic
 
+COMPONENT_PREP_ACTION = ( cd $(@D); \
+	$(GSED) -ri 's:.*(AUX_MODULES.*valid):\1:' modules.cfg && \
+	$(GSED) -r 's:.*(\#.*SUBPIXEL_RENDERING) .*:\1:' \
+		-i include/freetype/config/ftoption.h ) 
```

Comment:
It enables better hinting of fonts. In any case the update of freetype itself, without this, affects how fonts are rendered.